actions/scan-packages/action.yaml

```diff
@@ -28,7 +31,18 @@ runs:
         if [ "$scripts" != "null"  ]; then
           echo 'scripts=$scripts' >> "$GITHUB_OUTPUT"
         else
-          echo "::error::Error while reading or parsing the JSON. Couldn't find a script section"
+          echo "::error::Couldn't find a script section in package.json"
+        fi
+
+        node-version=$(cat package.json| jq -r '.engines["node"]')
+        if [ "$node-version" != 'null' ]; then
+          echo 'node-version-file=package.json' >> "$GITHUB_OUTPUT"
+        else if [ -f .node-version ]; then
+          echo 'node-version-file=.node-version' >> "$GITHUB_OUTPUT"
+        else if [ -f .nvmrc ]; then
+          echo 'node-version-file=.nvmrc' >> "$GITHUB_OUTPUT"
+        else
+          echo "::error::Couldn't find node version. Please specify using either package.json, .node-version or .nvmrc"
```

Comment:
I think it would actually be better to only allow for one configuration. Less complex in the workflow, but also in documentation

Reply:
It all comes down to what the team is using to set their environment. The engines field is only advisory, nvm or nodenv doesn't read that value. But that value has to be correct because that is what is communicated externally. nvm and nodenv only reads their on config files, .nvmrc and .node-version. So based on that the engine field in package.json should be used by the workflow since that is the only one that matters in a release